sentences/sl/light_HassTurnOff.yaml

```diff
@@ -2,7 +2,15 @@ language: sl
 intents:
   HassTurnOff:
     data:
-      - sentences: []
+      - sentences:
+          - "<izključi> [luč[i|ke|ko|ki]|svetilo|razsvetljavo|svetilko] [v|na] [sobi|prostoru] <area>"
```

Comment:
Change to round brackets [luč[i|ke|ko|ki]|svetilo|razsvetljavo|svetilko]

Reply:
You should do it in all places where sentence like "action domain state" is used. Action is turn on-off/increase-decrecrese brighhtnes etc. Domain is ligh/lamp etc. State id on/off, pecetntage etc.  So, all 3 parts are needed in (). Anything in between is optional un [].

Reply:
Thanks @spuljko , I'm working on it now. Had to translate all tests folder in order to run tests...I see there are still sentences to be fixed.